purchase_sale_link_by_origin/models/sale_order.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2023 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    @api.depends("origin")
+    def _compute_purchase_order_count(self):
+        return super()._compute_purchase_order_count()
+
+    def _get_purchase_orders(self):
+        po_related = self.env["purchase.order"]
+        for order in self:
+            po_related |= self.env["purchase.order"].search(
+                [("origin", "ilike", order.name)]
```

Comment:
```suggestion
                [("origin", "=", order.name)]
```
With ilike, all similar purchase order will also be found. Case:
Sale Order S000007, Purchase order with Source document S0000712 will also be included.

Reply:
Hi @AurelijaNorvaisaite, Thank you for your comment. The suggested correction has been made.